package.json

```diff
@@ -11,6 +11,7 @@
     "tap-spec": "4.1.1",
     "tape": "4.8.0"
   },
+  "homepage": "https://github.com/HenrikJoreteg/redux-bundler-preact",
```

Comment:
Maybe it's me, but I'm always linking to packages' source from npmjs.com. They consume this field to populate that link.